tests/highlighter/test_highlighter.py

```diff
@@ -56,6 +56,32 @@ def test_highlighter_highlight_pdf(
         num_highlights = sum([1 for page in doc for annotation in page.annots()])
         assert num_highlights == number_of_tokens_to_highlight
 
+def test_highlighter_redact_pdf(
+    invoice_predictions, invoice_ocr_obj, highlighter_pdf_path
+):
+    highlight = Highlighter(invoice_predictions, highlighter_pdf_path)
+    highlight.collect_tokens(invoice_ocr_obj.token_objects)
+    with tempfile.NamedTemporaryFile(suffix=".pdf") as f:
+        highlight.redact_pdf(f.name, invoice_ocr_obj.page_heights_and_widths)
+        doc = fitz.open(f.name)
+        num_remaining_redact_annot = sum([1 for page in doc for annotation in page.annots(types=[fitz.PDF_ANNOT_REDACT])])
+        assert num_remaining_redact_annot == 0
```

Comment:
Confirmed through print statements in redact_pdf that the correct number of redact annotations were made before they were executed and deleted. Should we make redact_pdf return the number of redact annotations made so that we can use that for tests?

Reply:
interesting question- in the tests it is also possible to read in the PDF and check the redactions I think (this is how we test highlighting)- have you found that not to be the case with how you've implemented it? I agree it would be easier, but also seems a bit odd

Reply:
per: https://github.com/pymupdf/PyMuPDF/issues/434. Apparently, Page.apply_redactions() deletes all redact annotations, which is why we are testing if the number of remaining annotations is 0. While this check is somewhat sound, it obviously doesn't exactly test if the redact went through.

Reply:
nevermind...I was calling apply_redactions which was deleting the redact annotations.

Reply:
Cool- I think we can return the number of redactions made then, just update the docstring to say that's what's happening